x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -473,8 +473,11 @@ export type PolicyInfo = Immutable<{
   id: string;
 }>;
 
-export type HostInfo = Immutable<{
-  metadata: HostMetadata;
+// Host Information as return by the Host Details API.
+// NOTE:  `HostInfo` type is the original and defined as Immutable. If needing to
+//        work with data that is not mutable, use `HostInfo` instead
```

Comment:
This message is confusing. It sounds like you are telling me to use `HostInfo` instead of `HostInfo`. 🤔

Reply:
yeah, this does not sound like a clear message. I'll change it in my next PR or this one if I end up making another change.
Thanks for the feedback.